README.md

```diff
@@ -138,3 +138,24 @@ unicorn -p 3001 -c ./config/unicorn.rb
 or whatever.
 
 This project rocks and uses MIT-LICENSE.
+
+
+### Publishing
+#### Credentials
+You'll need a Github personal access token with the correct scopes: `repo`, `write:packages`, `delete:packages`.
+
+You can create one here: https://github.com/settings/tokens
+
+Add this token to a local file at `~/.gem/credentials`:
+```
+:github: Bearer your_token
+```
```

Comment:
We shouldn't need this section since the gem is on RubyGems: https://rubygems.org/gems/clickfunnels_auth

```suggestion
```